testbed/go.mod

```diff
@@ -62,8 +62,9 @@ require (
 	github.com/Azure/go-autorest/autorest/validation v0.3.1 // indirect
 	github.com/Azure/go-autorest/logger v0.2.1 // indirect
 	github.com/Azure/go-autorest/tracing v0.6.0 // indirect
+	github.com/DataDog/datadog-agent/pkg/trace v0.42.0 // indirect
```

Comment:
The exporter is using this as of #18231:

```
github.com/DataDog/datadog-agent/pkg/trace v0.43.0-rc.3.0.20230201114415-fae4332beb98 // indirect
```

Do you think there's a way to get the same dep chain as the exporter?

Reply:
Sure thing!